src/Adapter/Driver/IbmDb2/Connection.php

```diff
@@ -262,8 +261,10 @@ public function execute($sql)
 
     /**
      * {@inheritDoc}
+     *
+     * @return null|string
      */
-    public function getLastGeneratedValue($name = null)
+    #[Override] public function getLastGeneratedValue($name = null): string|null
```

Comment:
Didn't we have a discussion last week or so about nullable return types being, for example, `?string`.